### PR TITLE
dynamic groups e2e

### PR DESCRIPTION
**Pull request by sashankaryal**
## What changes are proposed in this pull request?

E2E tests for dynamic groups

Video of tests running: https://github.com/voxel51/fiftyone/assets/66688606/5ef165c7-a7bb-4776-9a2b-9543926272a1

## How is this patch tested? If it is not, please explain why.
Locally.

```
yarn start-test-server &
yarn start
```

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

### What areas of FiftyOne does this PR affect?

-   [x] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: app/packages/looker/src/elements/common/tags.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ import { lookerTags } from "./tags.module.css";
 interface TagData {
   color: string;
   title: string;
+  path: string;
```
Comment (lanzhenw) [inline app/packages/looker/src/elements/common/tags.ts:43]: Nice~ 
I saw that you were using the tags to validate tooltip on hover. 
if we want to verify the tags/detections are correct on looker and on flash light, in that case, would screenshots be better? Or this is still the preferred pattern?

File: e2e/cypress/e2e/dynamic-groups/dynamic-groups.cy.ts [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    .should("have.length", fields.length);
+  fields.forEach((field) => {
+    const fieldWithDotEscaped = field.replace(/\./g, "\\.");
+    cy.get(`[data-cy=selector-result-${fieldWithDotEscaped}]`).should(
+      "be.visible"
+    );
+  });
+};
+
+const verifySceneIdTimestamp = (
```
Comment (lanzhenw) [inline e2e/cypress/e2e/dynamic-groups/dynamic-groups.cy.ts:21]: This is neat!

**Review by lanzhenw — APPROVED** [review]
LGTM

File: app/packages/looker/src/elements/common/tags.ts (hunk 1 above)
Comment (sashankaryal) [reply app/packages/looker/src/elements/common/tags.ts:43]: Rule of thumb is to reserve screenshots only for elements that are not in the DOM.

**Pull request merged**